wds-hero-widget.php

```diff
@@ -249,8 +252,8 @@ function wds_hero( $args = array(), $content = false ) {
 							$alt = $details['alt'];
 						?>
 
-							<div class="slider">
-								<img src="<?php echo $src; ?>" alt="<?php echo $alt; ?>" />
+							<div class="slider" style="background: url(<?php echo $src; ?>) no-repeat; background-size: cover;">
```

Comment:
That was an artifact of some testing I was doing locally on gaps below images in mobile. I've rolled that back and replaced it with your original code.